libs/barista-components/secondary-nav/src/secondary-nav.scss

```diff
@@ -108,84 +149,72 @@ $dt-secondarynav-header-padding-left: 20px;
 }
 
 .dt-secondary-nav-link {
-  color: $gray-700;
-  font-size: 14px;
-  line-height: 30px;
-  display: block;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
   text-decoration: none;
+  outline: none;
+  height: 56px;
+  font-size: 15px;
```

Comment:
This seems like an oddly specific size. Could we think about streamlining the font size for this one, so it fits the other components as well. Maybe the default 14px will also be alright.

Reply:
fixed, wrong font size was being used